### PR TITLE
Richard Davies Thursday July 1

### PR DESCRIPTION
**Pull request by edennoelh**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/updated/01-journal.adoc [hunk 1]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
-Dear husband be faithful to God, whatever becomes of me.
-The soldier seeing her, retreated back.
-So I went down to the troop of horse, standing in the street before my house.
-There was among them one bad man, who ran away in my father`'s debt,
+My wife and I foresaw that I would be sent there with them,
+though I was never a soldier, nor bore any arms for either side.
+And in a little time a troop of horsemen came for me, to bring me to prison.
+My relations offered to give bail for me, but it was not accepted,
+so I went to inform my wife of it, and to make myself ready to go with them.
+One of the soldiers came up the stairs after me, with a pistol and a naked sword,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: lets change naked sword to **unsheathed sword**

File: journal/updated/01-journal.adoc [hunk 2]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
+On my way I avoided the house of my uncle, a justice of the peace in this county,
+and brother-in-law to the captain before-mentioned,
+lest he should stop me from going to prison.
+
+So I went there alone, and told the marshal, John Mason,
+that I had come as prisoner; and he took me up to an upper garret for my lodging.
+But I had freedom of the whole house, as did the other prisoners,
 there being many Presbyterians, Independents, and Baptists,
-who were formerly my great acquaintance; but now they appeared very strange,
+who were formerly very acquainted with me; but now they appeared very strange,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: maybe: who were formerly my close acquaintances

File: journal/updated/01-journal.adoc [hunk 3]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
 and would not discourse with me.
-I considered the reason, and was informed,
-that these old formal church-members or professors had agreed
+I wondered at the reason for this, and was informed
+that these old, formal church-members or professors had agreed
 among themselves that they would not discourse with me,
 nor receive any books from me,
-lest the most serious inward Christians among them should turn Quakers.
+lest the most inwardly serious Christians among them turn Quakers.
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: I think we should leave this the way it was, but add the word "and". LIke this: lest the most serious **and** inward Christians among them should turn Quakers.

File: journal/updated/01-journal.adoc [hunk 4]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
-to live in it, and obey it; her name was Margaret Bowen, wife of John Bowen of Colfryn,
-and mother to Peter Bowen, in Martin`'s-le-Grand, London.
+and to live in it and obey it. Her name was Margaret Bowen,
+wife of John Bowen of Colfryn, and mother to Peter Bowen,
+in Martin`'s-le-Grand, London.
 
-In a little time my service was over among those professors in that prison,
-and the Lord made way for my enlargement.
-In about two weeks I came away, and left the rest of the prisoners there,
+In a little time, my service was over among those professors in that prison,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: **the** professors (sounds better I think)

File: journal/updated/01-journal.adoc [hunk 5]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
+who had received the truth some time before at Montgomery. Her name was Ann Hamon,
+wife of Thomas Hamon; and my wife and I went to visit her.
 
-About the year 1661, I went to a meeting at Edgemont near Wem, in Shropshire.
+About the year 1661, I went to a meeting at Edgemont, near Wem, in Shropshire.
 Here our friend William Gibson and I were taken prisoners,
 with about twenty-five or twenty-six more, and sent to Shrewsbury.
 We found the temper of the jailer to be very cruel.
-He threatened us with a great deal of hardship, if we did not eat of his food,
+He threatened us with a great deal of hardship if we did not eat of his food,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: Let's tweak this to say: if we did not eat his food, drink his drink, lie on his beds, and give him what he demanded.

File: journal/updated/01-journal.adoc [hunk 6]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
 for we suffered for conscience-sake towards God,
 and dared not break the command of Christ and the apostle,
-who commanded us not to swear at all.
-For our supposed transgressions was not only for meeting together,
-but for refusing the oaths of allegiance and supremacy.
-The jailer being a very passionate inconsiderate man,
-would go out in the morning from his own house, and not come back till night,
+who commanded us not to swear at all (for our supposed transgression
+was not only for meeting together,
+but for refusing the oaths of allegiance and supremacy).
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: let's add: refusing **to take** the oaths...

File: journal/updated/01-journal.adoc [hunk 7]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
-I desired him to let them have liberty to go to their friends in town at night,
-and to come there in the morning:
-and if he would not be pleased to grant them a little straw,
-then to let them go lie in their own beds; which he surlily denied,
+but now they were content for Christ`'s and the gospel`'s sake,
+to suffer that hardship. I desired him to let them have
+liberty to go to their friends in town at night,
+and to return in the morning;
+and if he wasn`'t willing to grant them a little straw,
+then to let them go lie in their own beds; which he sullenly denied,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: let's change sullenly to roughly

File: journal/updated/01-journal.adoc [hunk 8]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
+I asked him whether he knew that I also was a prisoner there with my friends;
+and he asked me whether I slept upon the boards with them; I told him I did.
+He said he was sorry for it, but went away in anger, being much discontented in himself.
+
+I did not see him again till next morning, at which time I went to him again,
+and discoursed kindly with him. He said he had enquired about me in town,
+and that I might have the liberty of the town if I wished.
+I acknowledged his kindness, but told him it would be no comfort to me
+to have the liberty of the town, and leave my friends and brethren there in prison.
+He said I might stay there in prison with them, then.
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: I'm not sure the ", then" at the end works. Maybe we can change it to a quotation, and say: He said, "Then you may stay there in the prison with them."

File: journal/updated/01-journal.adoc [hunk 9]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
-prevailed with him to let them go,
-and he took our words for our appearance at the next court session.
-
-Through the goodness of God, we all returned together to Shrewsbury, to our prison,
+to see whether we could be granted the permission
+to go home till the next court session; but it was not then granted,
+although he told me that if I pleased I might go home till then.
+I told him he might as freely let them go as me, for most of them lived in the county,
+and I lived out of the county; but we could get nothing more from him at that time.
+I was uneasy in myself to leave him alone, seeing he had a regard for me,
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: Let's reword this. Maybe: Seeing he had a regard for me, I felt uneasy in myself to let it drop,

File: journal/updated/01-journal.adoc [hunk 10]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
+and so continued to press for my friends`' further liberty.
+
+A little time afterwards, by the argument that it was harvest time,
+and hard for the mens`' wives, or some of their families,
+to obtain their weekly necessities, I, with my friend John Millington,
+prevailed with him to let us go,
+him taking our word that we would appear at the next court session.
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: **he** taking our word

File: journal/updated/01-journal.adoc [hunk 11]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
 When we came to Bridgenorth,
-we were put in a large spacious room in the house of correction,
-to be there in the day time, that we might be all together, and ready when called for;
-but we had liberty of going in and out for lodging, and what necessaries we needed;
-no keeper being over us, but what we set ourselves to look to the door,
-and that too many Friends might not be out at once,
-and those were not to stay out too long.
-We saw it was convenient, that Friends should go out by two and two, to walk the streets;
+we were put in a large, spacious room in the house of correction.
+We arrived there in the day time, so that we might all be together and ready when called for;
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: The idea is: we were put in a large, spacious room in the house of correction, **in order to be there during the day time,** that we we might all be together and ready when called for...

File: journal/updated/01-journal.adoc [hunk 12]
```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
...
-who appeared something like a gentleman, and asked me, whether he might see the Quakers;
-I desired him to walk up along with me, and he should see them.
-When I had brought him up to the room where Friends were walking, I told him,
-those were they.
-He answered, these be Christians like ourselves, but where are the Quakers?
-I told him, these were they that were called Quakers.
-He asked me, whether I was one of them; I told him, I was one so called.
+Once it fell to my lot to be at the door. The door was always open,
+so that those who desired to do so
+might come and see us (several of whom we had reasonings and
```
Comment (Henderjay) [inline journal/updated/01-journal.adoc]: this needs the "with".  (**with** several of whom

**Review by Henderjay — CHANGES_REQUESTED** [review]
few things to tweak 👍🏻

File: journal/updated/01-journal.adoc (hunk 12 above)
Comment (edennoelh) [reply journal/updated/01-journal.adoc]: It does have the with, just later in the sentence. It sounded better to me that way, but I can change it if you want.

**Comment by Henderjay** [comment]
merged

**Pull request merged**